### PR TITLE
List Button (Wearable): add examples and update styles

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/954
[Problem] There is no examples presenting buttons on list
[Solution] add examples and update styles

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

**Comment by grzegorz-czajkowski** [comment]
I am not sure about press effect (Type 1, Style 2). Since VI does not specify it. I can find video only.

File: examples/wearable/UIComponents/contents/presentation-views/list/list-button/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	<div class="ui-page" data-go-to-top-button="true">
+		<header>
+			<h2 class="ui-title">
+				List Button
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li>
+					<a href="three-buttons.html">
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/presentation-views/list/list-button/index.html:27]: I think more natural is to have Two buttons example first in index

File: examples/wearable/UIComponents/css/style.css [hunk 2]
```diff
@@ -375,14 +375,13 @@ li.static::before {
...
-}
-.app-icon-delete::before {
-	background-image: url(../contents/list/images/tw_list_delete_holo_dark.png);
+.list-button-color::before {
 }
-.app-icon-setting::before {
-	background-image: url(../contents/list/images/tw_list_setting_holo_dark.png);
+
+.app-icon::before {
+	/* TODO: replace this icon with IC icon accorfing to the guide */
```
Comment (lslachciak) [inline examples/wearable/UIComponents/css/style.css]: typo: accorfing --> according

**Review by lslachciak — CHANGES_REQUESTED** [review]
works fine, two minor comments in code

File: examples/wearable/UIComponents/contents/presentation-views/list/list-button/index.html (hunk 1 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/list-button/index.html:27]: List button examples in guide are presented in such order (see both guides html and pdf). How about leving this as it is?

File: examples/wearable/UIComponents/css/style.css (hunk 2 above)
Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/css/style.css]: Done.

**Review by lslachciak — APPROVED** [review]

**Pull request merged**